src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,33 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health) {
+    this.name = name;
+    this.health = health || 100;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health, hidden) {
```

Comment:
```suggestion
  constructor(name, health) {
```
I think hidden is not necessary here, that belongs only to Herbivore class